website/docs/language/settings/index.html.md

```diff
@@ -31,11 +31,20 @@ following sections.
 
 ## Configuring a Terraform Backend
 
-The nested `backend` block configures which backend Terraform should use.
+The nested `backend` block configures which state backend Terraform should use.
 
 The syntax and behavior of the `backend` block is described in [Backend
 Configuration](/docs/language/settings/backends/configuration.html).
 
+## Configuring Terraform Cloud
+
+The nested `cloud` block configures Terraform Cloud for enabling its
+[CLI-driven run workflow](/docs/cloud/run/cli.html).
+
+The syntax and behavior of the `cloud` block is described in [Configuring
+Terraform Cloud](/docs/cli/configuring-terraform-cloud/index.html) in the
+Terraform CLI documentation.
```

Comment:
The choice of link here ends up being disorienting — we _have_ a summary of the block syntax right here in the language docs, and that's what you'd expect the main link on this overview page to be, but it links out to the CLI docs instead. 

Some of this is just that it's legit confusing where this documentation belongs — it's the one piece where there's a good argument for putting it in any of the three main sections. But anyway, to make this a bit more predictable, I suggest using bullet points to make it more clear which links stay local and which ones go elsewhere:

```suggestion
- For a summary of the `cloud` block's syntax, see 
[Terraform Cloud Configuration](/docs/language/settings/terraform-cloud.html).

- For more complete details about configuring and using Terraform Cloud with Terraform CLI,
see [Configuring Terraform Cloud](/docs/cli/configuring-terraform-cloud/index.html) in the
Terraform CLI documentation.
```

Reply:
Let's meet later on this (as we already need to, to discuss the larger changes); TBH I left the language/settings page there only because it 'makes sense' to leave some sort of reference to the block being a `terraform` options block, but it's _only_ to advertise the canonical documentation of this block, which I believe should be in "The" documentation on this feature, which I believe should be in the CLI documentation :D 

I guess under that light this exact change is fine, though the point I'm getting it is 'The thing here in language/settings is actually sort of worthless and only a placeholder, so let's just only mention the CLI docs 🤷‍♂️ '